packages/volto/src/config/Widgets.jsx

```diff
@@ -164,10 +165,29 @@ export const widgetMapping = {
   },
 };
 
+export const widgetValidators = {
+  object_list: {
+    method: objectListValidator,
+    name: 'object_list_validator_iternal_items',
+  },
+};
+
 // Default Widget
 export const defaultWidget = TextWidget;
 
 export function installDefaultWidgets(config) {
   config.widgets = widgetMapping;
   config.widgets.default = defaultWidget;
+
```

Comment:
I don't understand the intention of this loop... Can we register it in an unitary way like the others and where the other validators are defined? 

https://github.com/plone/volto/blob/95012fc5f8804982c6ea4e65e81efc2de163b204/packages/volto/src/config/validation.ts